src/__test__/authorize.test.ts

```diff
@@ -101,3 +101,77 @@ describe('authorize - with secret as callback in options', () => {
     })
   })
 })
+
+describe('authorize - with onAuthentication callback in options', () => {
+  let token: string = ''
+  let tokenWrong: string = ''
```

Comment:
We could name this variable : `wrongToken` instead of `tokenWrong`.
It is better for semantic, what do you think ?